Sprint-3/3-mandatory-practice/implement/repeat.test.js

```diff
@@ -16,6 +16,35 @@ test("should repeat the string count times", () => {
     expect(repeatedStr).toEqual("hellohellohello");
     });
 
+    
+    test("should return the original string when count is 1", () => {
+        const str = "hello";
+        const count = 1;
+        const repeatedStr = repeat(str, count);
+        expect(repeatedStr).toEqual("hello");
+    });
+    
+    
+    test("should return an empty string when count is 0", () => {
+        const str = "hello";
+        const count = 0;
+        const repeatedStr = repeat(str, count);
+        expect(repeatedStr).toEqual("");
+    });
+    
+
+    test("should throw an error when count is negative", () => {
+        const str = "hello";
+        const count = -2;
+        try {
+            repeat(str, count);
+        } catch (e) {
+            expect(e.message).toEqual("Count must be a non-negative integer.");
```

Comment:
Using a try/catch here definitely works well. As a stretch, you might want to look at the matcher [`toThrow`](https://jestjs.io/docs/expect#tothrowerror) here, and maybe rewriting this to use that.